app/src/pages/explorer/sample/Sample.vue

```diff
@@ -0,0 +1,9 @@
+<template src="./sample.html"></template>
+<script src="./sample.js"></script>
+
+<style>
+.sample-view {
+  width: 100%;
+  height: 100%;
+}
```

Comment:
Do we need this, would you please check if this has any effect?

Reply:
my mistake I forgot to remove it